<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewSparePart</class>
 <widget class="QDialog" name="ViewSparePart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация о запчасти</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Производитель</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Совместимость с автомобилями</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название запчасти</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Оригинал / аналог</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="isOriginalLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="sparePartNameLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="autoCompatibilityLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="quantityInStockLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="priceLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Количество в наличии</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="manufacturerLine">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1px solid gray;
    border-radius: 7px;
    background-color: #dbdcf7;
	padding-left: 3px;
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Цена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
